user_manual/groupware/calendar.rst

```diff
@@ -211,7 +218,20 @@ An event may be set as "recurring", so that it can happen every day, week, month
 You can also tell when the recurrence ends.
 
 .. figure:: images/calendar_event_repeat.png
+              :scale: 80%
+
+.. _calendar-trash-bin:
+
+Trash bin
+~~~~~~~~~
+
+If you delete events, tasks or a calendar in Calendar, your data won't be gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period of 30 days those items will be deleted permanently. You can also permanently delete earlier if you wish.
+
+.. figure:: images/calendar_trash_bin.png
+
+The ``Empty trash bin`` buttons will wipe all trash bin contents in one step.
 
+.. tip:: The trash bin is only accessible from the Calendar app. Any connected application or app won't be able to display its contents. However, events, tasks and calendars deleted in external applications will also end up in the trash bin.
```

Comment:
Precise Nextcloud Calendar app.

> Application or app?

Maybe client or app?

Reply:
the whole page is about the *Calendar app*

Reply:
Please change `connected application or app` though